setup.py

```diff
@@ -84,7 +84,7 @@
             'ansible_playbooks = cfme.ansible.playbooks:PlaybooksCollection',
             'ansible_repositories = cfme.ansible.repositories:RepositoryCollection',
             'datastores = cfme.infrastructure.datastore:DatastoreCollection',
-            'service_dialogs = cfme.automate.service_dialogs:DialogCollection',
+            'service_dialogs = cfme.automate.dialog_collection_pick:collection_pick',
```

Comment:
**REQUIRED:** When does this get resolved?

Reply:
This is implemented . Version pick on collection is implemented by Milan.